docs/source/deployorderer/ordererchecklist.md

```diff
@@ -0,0 +1,307 @@
+* **`ClientPrivateKey`**: Provide the path to and filename of the private key generated by your TLS CA for this node.
+
+In general, these four parameters would only need to be configured if you want to configure multiple listeners for your orderers. In most circumstances a single listener is sufficient. Either set these four parameters together or leave them unset.
+
+* **`ListenPort`**
+* **`ListenAddress`**
+* **`ServerCertificate`**
+* **`ServerPrivateKey`**
+
+Note: the `ServerCertificate` and `ServerPrivateKey` must be set to the same values as the `General.TLS.PrivateKey` and `General.TLS.Certificate`.
+
+## General.BoostrapMethod
+
+```
+# Bootstrap method: The method by which to obtain the bootstrap block
+# system channel is specified. The option can be one of:
+#   "file" - path to a file containing the genesis block or config block of system channel
+#   "none" - allows an orderer to start without a system channel configuration
+BootstrapMethod: file
+```
+
+* **`BootstrapMethod`**: (default value should not be overridden) Unless you plan to use a file type other than “file”, this value should be left as is.
+
+## General.BoostrapFile
+
+```
+# Bootstrap file: The file containing the bootstrap block to use when
+# initializing the orderer system channel and BootstrapMethod is set to
+# "file".  The bootstrap file can be the genesis block, and it can also be
+# a config block for late bootstrap of some consensus methods like Raft.
+# Generate a genesis block by updating $FABRIC_CFG_PATH/configtx.yaml and
+# using configtxgen command with "-outputBlock" option.
+# Defaults to file "genesisblock" (in $FABRIC_CFG_PATH directory) if not specified.
+BootstrapFile:
+```
+
+* **`BoostrapFile`**: (default value should be overridden) Specify the location and name of the ordering service genesis block to use when this node is created.
+
+## General.LocalMSPDir
+
+```
+# LocalMSPDir is where to find the private crypto material needed by the
+# orderer. It is set relative here as a default for dev environments but
+# should be changed to the real location in production.
+LocalMSPDir: msp
+```
+
+**`LocalMSPDir`**: (default value should not be overridden) This is the path to the ordering node's local MSP, which must be created before it can be deployed. The path can be absolute or relative to `FABRIC_CFG_PATH` (by default, it is `/etc/hyperledger/fabric` in the orderer image). Unless an absolute path is specified to a folder named something other than "msp", the ordering node defaults to looking for a folder called “msp” at the path (in other words, `FABRIC_CFG_PATH/msp`) and when using the orderer image: `/etc/hyperledger/fabric/msp`. If you are using the recommended folder structure described in the [Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html) topic, it would be relative to the `FABRIC_CFG_PATH` as follows:
```

Comment:
This will often be overridden.
This paragraph seems to have 2x the number of words as needed :-)

Reply:
Given that cert stuff like this is the thing people seem to most often screw up, I don't mind oversharing.